seriously-simple-podcasting.php

```diff
@@ -56,9 +57,10 @@
  * @todo the admin_controller should really be renamed, as it's not really 'admin' specific
  * @todo alternatively the non admin specific functionality should be moved into it's own 'foundation' controller, perhaps even the parent controller, or a trait
  */
-global $ssp_admin, $ss_podcasting;
+global $ssp_admin, $ss_podcasting, $ssp_player;
 $ssp_admin     = new Admin_Controller( __FILE__, SSP_VERSION );
 $ss_podcasting = new Frontend_Controller( __FILE__, SSP_VERSION );
+$ssp_player = new Players_Controller(__FILE__, SSP_VERSION);
```

Comment:
Small request, let's call this global $ssp_players, just to tie into the naming convention of the class.

Eventually we will refactor all these horrible globals, but this is fine for now.